www/src/components/markdown-page-footer.js

```diff
@@ -15,7 +15,7 @@ export default class MarkdownPageFooter extends React.Component {
         <hr sx={{ display: `none` }} />
         {this.props.page && (
           <a
-            sx={{ variant: `links.muted`, mt: 9 }}
+            sx={{ variant: `links.muted`, mt: 6 }}
```

Comment:
I'd stick to `9` here:

![image](https://user-images.githubusercontent.com/21834/68305464-0369ea00-00a8-11ea-885c-f12fcb4a7af9.png)

Reply:
Yes I agree. I'll change it back.

Reply:
👍 🙏